components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/dto/CertificateInformationDTO.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * you may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.apimgt.api.dto;
+
+/**
+ * This class represents the common parameters in a X509 certificate.
+ */
+public class CertificateInformationDTO {
+
+    private String status;
+    private String from;
+    private String to;
+    private String subject;
+    private String version;
+
+    public String getStatus() {
+
```

Comment:
why this class has too many spaces?

Reply:
This has happened when running the WSO2 Code formatting in Idea. It adds a newline before the first line in of a method.